commands-yml/commands/element/find-elements.yml

```diff
@@ -0,0 +1,93 @@
+---
+name: Find Elements
+short_description: Search for multiple elements
+description:
+  >
+     Get a list of elements that match the [locator selector](/docs/en/about-appium/getting-started).
+selector_strategies:
+  # TODO Document the rest
+  - name: Accessibility ID
+    description: Read a unique identifier for a UI element. For XCUITest it is the element's `resource-id` attribute. For Android it is the element's `content-desc` attribute.
+  - name: Class name
+    description: "For IOS it is the full name of the XCUI element and begins with XCUI. For Android it is the full name of the UIAutomator2 class (e.g.: android.widget.TextView)"
```

Comment:
Why is this in quotes? Also, the XCUITest part should start with `XCUIElementType`.